pootle/apps/import_export/utils.py

```diff
@@ -90,15 +90,13 @@ def revision(self):
         return revision.get(self.context.__class__)(
             self.context.directory).get(key="stats")[:10] or "0"
 
-    @property
-    def relative_path(self):
-        return "offline_tm/%s/%s" % (
-            self.context.language.code,
-            self.filename)
-
     def get_url(self):
         if self.exported_revision:
-            return urljoin(settings.MEDIA_URL, self.relative_path)
+            relative_path = "offline_tm/%s/%s" % (
+                self.context.language.code,
+                self.get_filename(self.exported_revision)
+            )
+            return urljoin(settings.MEDIA_URL, relative_path)
```

Comment:
im wondering if this wants to use urlparse.urljoin https://docs.python.org/2/library/urlparse.html#urlparse.urljoin